### ISSUE TITLE
[Question]: is 'navigation' to an anchor tag supported?

### ISSUE THREAD

**Issue by danielquintero** [jorgegorka/svelte-router#67]
I thought default anchor 'navigation' would work but what I tried so far did not work. Since the router listens to simple anchor tags with href, I am wondering if is there a way to do something like:

```
<a href="#Products">Products</a>
....
<a name="Products"> Products section</a>

thanks!

**Comment by jorgegorka** [comment]
Hi @danielquintero 

If you want to navigate to an anchor tag just add the url in the href like

`<a href="/my/page-address#Products>Products</a>`

**Comment by danielquintero** [comment]
@jorgegorka thanks for answering, unfortunately I keep being unable to get this simple thing to work.

I running on 5.6.0 of svelte-router-spa (latest)

and the smallest reproducible setup is:

src/routes.js
```
import PublicIndexTest from './views/public/index-test.svelte';
import PublicLayoutTest from './views/public/layout-test.svelte';

const routes = [
    {
        name: '/',
        component: PublicIndexTest,
        layout: PublicLayoutTest,
    },
];

export { routes };
```
src/views/public/layout-test.svelte
```
<script>
    import { Route } from 'svelte-router-spa';
    export let currentRoute;
</script>

<div class="container">

    <section>HEADER</section>

    <Route {currentRoute} />

    <section>FOOTER</section>

</div>
```

src/views/public/index-test.svelte
```
<div style="height: 100px; position: relative">
    <a href="#prices">Pricing</a> // <= THIS ANCHOR LINK WON'T WORK
</div>

<div style="height: 500px; background-color:#e5e5e5">ABOUT</div>
<div style="height: 600px; background-color:red">INFO</div>
<div style="height: 400px; background-color:yellow">
    PRICES
    <a name="prices">Prices Section</a> // <= SHOULD COME HERE
</div>
```

when forcing the browser url to be `localhost:5000/#prices` it works fine, so I am suspecting something prevents browser from it's normal behaviour?

Thanks again, any help is much appreciate it!

**Comment by danielquintero** [comment]
Note: I tried `<a href="/#prices">Prices</a>` and even having it on nested routes like you suggested

**Comment by jorgegorka** [comment]
Hi @danielquintero I've published version 5.7.1 that should not remove the anchor tag from the url.   Hopefully that will solve your issue.

**Comment by popoleeMaster** [comment]
Sorry for the "revive" but this doesn't looks like it works to me, using the same minimal exemple above
